Gemfile

```diff
@@ -161,9 +161,7 @@ group :test, :development do
   gem 'certified', require: false
   gem 'fabrication', require: false
 
-  # TODO: upgrading to 1.10.1 cause it breaks our test suite.
-  # We want our test suite fixed though to support this upgrade.
-  gem 'mocha', '1.8.0', require: false
+  gem 'mocha', '1.11.2', require: false
```

Comment:
Can we remove the pinned version completely, so that we get auto-PRs for new versions?

Reply:
doh, of course 😅